scripts/imgtool/image.py

```diff
@@ -246,14 +258,22 @@ def ecies_p256_hkdf(self, enckey, plainkey):
     def create(self, key, enckey, dependencies=None):
         self.enckey = enckey
 
-        if dependencies is None:
-            dependencies_num = 0
-            protected_tlv_size = 0
-        else:
-            # Size of a Dependency TLV = Header ('BBH') + Payload('IBBHI')
-            # = 16 Bytes
+        protected_tlv_size = 0
+
+        if self.security_counter is not None:
+            # TLV info header + security counter TLV
+            # Size of the security counter TLV: header ('HH') + payload ('I')
+            #                                   = 4 + 4 = 8 Bytes
+            protected_tlv_size = TLV_INFO_SIZE + TLV_SIZE + 4
```

Comment:
`TLV_INFO_SIZE` should be removed if changed as suggested in other comments...

Reply:
Done.